### DISCUSSION TITLE
Need help, iphone sent to PC, photo rename to FullSizeRender.jpg and can't open

### DISCUSSION THREAD

**Discussion by IvanXieMel** [localsend/localsend#559] [Discussion (English, International)]
The app was really good and simple
Today, I tried to use the iPhone app to send photos to my PC. But all the photos rename to FullSizeRender.jpg and can't open on PC. Is any update caused that? thanks

**Comment by IvanXieMel** [comment]
Anyone can help? thanks in advance

**Comment by zillydev** [comment]
Use the share option from the Photos app of your iPhone and it should send normally.
